src/install/dracut-install.c

```diff
@@ -1657,7 +1656,7 @@ static int modalias_list(struct kmod_ctx *ctx)
         for (FTSENT *ftsent = fts_read(fts); ftsent != NULL; ftsent = fts_read(fts)) {
                 _cleanup_fclose_ FILE *f = NULL;
                 _cleanup_kmod_module_unref_list_ struct kmod_list *list = NULL;
-                struct kmod_list *l;
+                struct kmod_list *l = NULL;
```

Comment:
This line is not even needed. The same variable is already declared at function scope.

Reply:
Correct, removed it.